images/build/debian-iptables/distroless/get-binaries-from-package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+get_binary_list() {
+BIN_LIST="$(dpkg -L ${1} | grep -E '/s?bin/' |sed 's/\n/ /g')"
+	for binary in $BIN_LIST; do
+		/stage-binary-and-deps.sh "${binary}" "${2}"
+	done
+}
+
+main() {
+    	local PACKAGE=$1
+    	local STAGE_DIR="${2}/"
```

Comment:
`local` is not POSIX sh / not really portable.

Reply:
Question on this: as we already rely on bash to run the other scripts, maybe should I just change this to bash as well as we are not actually porting this script to run inside distroless, but inside the builder image?